docs/source/ucx.rst

```diff
@@ -1,35 +1,87 @@
-- ``DASK_UCX__NVLINK=True`` (default: ``False``): Adds ``cuda_ipc`` to ``UCX_TLS`` -- required for all NVLink transfers in UCX, only affects intra-node.
-- ``DASK_UCX__INFINIBAND=True`` (defalt: ``False``): Adds ``rc`` to ``UCX_TLS`` -- required for all InfiniBand transfers in UCX, only affects inter-node.
-- ``DASK_UCX__RDMACM=True`` (default: ``False``): Replaces ``sockcm`` by ``rdmacm`` in ``UCX_TLS`` and ``UCX_SOCKADDR_TLS_PRIORITY``. ``rdmacm`` is the recommended method by UCX to use with IB and currently won't work if ``DASK_UCX__INFINIBAND=False``.
-- ``DASK_UCX__NET_DEVICES=mlx5_0:1`` (default: ``None``, causes UCX to decide what device to use, possibly being suboptimal, implies ``UCX_NET_DEVICES=all``): this is very important when ``DASK_UCX__INFINIBAND=True`` to ensure the scheduler is connected over the InfiniBand interface. When ``DASK_UCX__INFINIBAND=False`` it's recommended to use the ethernet device instead, e.g., ``DASK_UCX__NET_DEVICES=enp1s0f0`` on a DGX-1.
-- ``DASK_RMM__POOL_SIZE=1GB``: allocates an RMM pool for the process. In some circumstances, the Dask scheduler will deserialize CUDA data and cause a crash if there's no pool.
+    Adds ``rc`` to ``UCX_TLS``, enabling InfiniBand transfers over UCX; affects inter-node communication only.
+
+
+- ``ucx.rdmacm: true`` -- recommended for InfiniBand.
+
+    Replaces ``sockcm`` with ``rdmacm`` in ``UCX_TLS`` and ``UCX_SOCKADDR_TLS_PRIORITY``, *enabling remote direct memory access (RDMA) for connection management.*
+    This is recommended by UCX for use with InfiniBand, and will have no effect if InfiniBand tranfers are disabled.
+
+- ``ucx.net-devices: <str>`` -- recommended.
+
+    Explicitly sets ``UCX_NET_DEVICES`` instead of defaulting to ``"all"``, which can result in suboptimal performance.
+    If using InfiniBand, set to the desired IB device, e.g. ``"mlx5_0:1"``.
+    If InfiniBand is disabled, set to the ethernet device, e.g. ``"enp1s0f0"`` on a DGX-1.
+    All available UCX-compatible devices can be listed by running ``ucx_info -d`` or ``ifconfig`` *(are all the options under ifconfig compatible)*.
+
+- ``rmm.pool-size: <str | int>`` -- recommended.
+
+    Allocates an RMM pool of the specified size for the process; size can be provided with an integer number of bytes or in human readable format, e.g. ``"4GB"``.
+    In addition to reducing the cost of mapping incurred by memory transfers, a pool can prevent the Dask scheduler from deserializing CUDA data and causing a crash.
+    It is recommended to set the pool size to at least the minimum amount of memory used by the process; if possible, one can map all GPU memory to a single pool, to be utilized for the lifetime of the process.
+
+
+.. note::
+    These options can also be used with mainline Dask/Distributed.
+    However, this may disable a variety of features, such as *put some features exclusive to Dask-CUDA here*. 
```

Comment:
`net-devices="auto"` is an exclusive feature of Dask-CUDA, I don't think there are others that apply to this section. The line below this should also be clarifying in this context.

Reply:
Yeah, so far from the config section alone I have:

- Use of CUDA transfers for task on GPU
- Setting `net-devices="auto"`

Anything I'm missing here?

Reply:
> Use of CUDA transfers for task on GPU

This is again not true. A user can very much use mainline Dask with GPUs, they will only not benefit from the specializations Dask-CUDA provides -- thus my comment on the line that is immediately below this clarifying that difference. Therefore, the only thing I can think of here is indeed `net-devices="auto"`.